### DISCUSSION TITLE
What does 30% utilization mean?

### DISCUSSION THREAD

**Discussion by scottyeager** [threefoldtech/minting_v3#7] [General]
Continuing discussion from [here](https://github.com/threefoldtech/tfchain/issues/176#issuecomment-1016771110). How do we calculate 30% utilization to unlock tokens?

**Comment by LeeSmet** [comment]
As said, "30%" is not specified currently. There is currently no exact specification of what this means.

**Comment by scottyeager** [comment]
Indeed, that's the idea of this issue, to come to an agreement on what the spec should be.
